src/otx/config/mmdeploy/base/base_detection_dynamic.py

```diff
@@ -0,0 +1,22 @@
+"""Detection models dynamic deploy config."""
+
+_base_ = ["./base_detection_static.py"]
+
+ir_config = dict(
+    dynamic_axes={
+        "image": {
```

Comment:
We don't need dynamic input for our models

Reply:
ok I'll remove it.

Reply:
I have a question. If resize type is set as `fit_to_window`, I think input size will be dynamic. Then if then, are these values necessary?

Reply:
MAPI should use a fixed input shape, as far as I know, so we definitely don't need dynamic h,w here.
If you think that's not the case, and MPAI's resize functions can output variable sizes, let's review this together